docs/storybook/src/Decorators.tsx

```diff
@@ -40,3 +55,44 @@ export const InitializerDecorator: Decorator = (Story) => {
   if (!initialized) return <>Initializing...</>;
   return <Story />;
 };
+
+// Decorator that forwards console.* calls to the Storybook actions panel
```

Comment:
This should be removed, https://storybook.js.org/docs/essentials/actions can be used directly.